packages/core/src/internal/clients/vector/AbstractVectorIndexClient.ts

```diff
@@ -133,6 +135,34 @@ export abstract class AbstractVectorIndexClient
     return await this.dataClient.search(indexName, queryVector, options);
   }
 
+  /**
+   * Searches for the most similar vectors to the query vector in the index.
+   *
+   * Ranks the vectors according to the similarity metric specified when the
+   * index was created.
+   * Also retursn the vectors associated with each result.
```

Comment:
Minor typo.

Reply:
Good catch. Fixed in [297d469](https://github.com/momentohq/client-sdk-javascript/pull/1034/commits/297d469e0351c00ac425082d52a40dfdace5b447)